### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by olark1d**
Додав і пул-реквест до HOMEWORK_6, нарешті, чекаю фідбек, та більше 0 балів)

### CODE REVIEW

File: HOMEWORK_6/EX1.CPP [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    int endNumber;
+    int stepNumber;
+
+    std::cout<<"Enter start number, step number and end number: ";
+    std::cin>>startNumber>>stepNumber>>endNumber;
+
+    if(startNumber > 0 && stepNumber > 0 && endNumber > 0)
+    {
+        std::cout<<"Arithmetic progression: \n";
+        for(int i = startNumber; i <= endNumber; i+=stepNumber)
```
Comment (bohdanpc) [inline HOMEWORK_6/EX1.CPP]: the last argument is not endNumber but more like numbersCount, that is the number of elements to output which is not the case in your impl.

File: HOMEWORK_6/EX2.CPP [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    std::cout<<"Enter the element number: ";
+    std::cin>>elementNumber;
+    
+    if(elementNumber > 0)
+    {
+        do
+        {
+            result = firstValue + secondValue;
+
+            if(index > 0)
```
Comment (bohdanpc) [inline HOMEWORK_6/EX2.CPP:20]: I just don't feel like this condition is a must-have... I can think of simpler solution

File: HOMEWORK_6/EX3.CPP [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    {
+        for(int i = 1; i <= inputNum; i++)
+        {
+            result *= i;
+        }
+
+        std::cout<<"Factorial of "<<inputNum<<" = "<<result<<std::endl; 
+    
+    }
+
```
Comment (bohdanpc) [inline HOMEWORK_6/EX3.CPP]: not needed empty line)

File: HOMEWORK_6/EX4.CPP [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        
+        for(int i = 1; i <= height; i++)
+        {
+            for(int j = 0; j < width3; j++)
+            {
+                std::cout<<(i+j) % 2;
+            }
+            std::cout<<std::endl;
+            width3++;
+        }
```
Comment (bohdanpc) [inline HOMEWORK_6/EX4.CPP:78]: It should start from 1 element and increase down by one each new line. Aka triangle, not trapezium

File: HOMEWORK_6/EX1.CPP (hunk 1 above)
Comment (olark1d) [reply HOMEWORK_6/EX1.CPP]: correct code in Ex1, I hope it's true now

File: HOMEWORK_6/EX2.CPP (hunk 2 above)
Comment (olark1d) [reply HOMEWORK_6/EX2.CPP:20]: I haven't no idea, how to do it(

File: HOMEWORK_6/EX4.CPP (hunk 4 above)
Comment (olark1d) [reply HOMEWORK_6/EX4.CPP:78]: found the missing piece of code and added it, but I don't see any problem with the highlighted code, below I will add screens of the task and the result I have
Снимок экрана 2024-05-18 001443
Снимок экрана 2024-05-18 001510

Comment (bohdanpc) [reply HOMEWORK_6/EX4.CPP:78]: My bad, thanks for the console output

File: HOMEWORK_6/EX2.CPP (hunk 2 above)
Comment (bohdanpc) [reply HOMEWORK_6/EX2.CPP:20]: int first = 0;
int  second = 1;
int summ = 0;

    if (elementNubmer == 0)
        return first;

    for (int i = 2; i <= elementNubmer; i++) {
        summ = first + second;
        first = second;
        second = summ;
    }
    return second;
}

**Pull request merged**